Update documentation (#240)
AY2021S1-CS2103T-F11-4/tp: docs/DeveloperGuide.md

## **Introduction**

Welcome! We are excited that you want to learn about <strong><i>TBM</i></strong>.<br>

> This page contains step-by-step instructions on how to get started in contributing or getting yourself involved in this awesome project! However, before you get too excited, allow us to introduce you to the core concepts that underpins the heart of <strong><i>TBM</i></strong>.

Reviewer: I think for `<strong><i>` we should follow the markdown conventions and use `**` and `_`?

According to https://se-education.org/guides/conventions/markdown.html

Author: Oh nice, I keep thinking markdown has no italics.

> This page contains step-by-step instructions on how to get started in contributing or getting yourself involved in this awesome project! However, before you get too excited, allow us to introduce you to the core concepts that underpins the heart of <strong><i>TBM</i></strong>.

Reviewer: The comments by Team 2 advices to link Travelling BusinessMan to TBM, maybe we can do that here since it is the first mention of TBM?

## **Appendix: Instructions for manual testing**

> Given below are instructions to test the app manually. Additionally, testers can look through [Testing guide](Testing.md) to get started as well on automated testing.

Reviewer 2: ```suggestion
Given below are instructions to test the app manually. Additionally, testers can look through our [Testing Guide](Testing.md) to get started on automated testing.
```
